plugins/woocommerce-admin/client/guided-tours/wc-addons-tour/get-config.ts

```diff
@@ -0,0 +1,206 @@
+/**
+ * External dependencies
+ */
+import { TourKitTypes } from '@woocommerce/components';
+import { __ } from '@wordpress/i18n';
+import { createElement, createInterpolateElement } from '@wordpress/element';
+
+export const getTourConfig = ( {
+	closeHandler,
+	onNextStepHandler,
+	onPreviousStepHandler,
+	autoScrollBlock,
+}: {
+	closeHandler: TourKitTypes.CloseHandler;
+	onNextStepHandler: ( currentStepIndex: number ) => void;
+	onPreviousStepHandler: ( currentStepIndex: number ) => void;
+	autoScrollBlock: ScrollLogicalPosition;
+} ): TourKitTypes.WooConfig => {
+	const lineBreak = createElement( 'br' );
+
+	return {
+		placement: 'auto',
+		options: {
+			effects: {
+				spotlight: {
+					interactivity: {
+						enabled: true,
+						rootElementSelector: '.woocommerce.wc-addons-wrap',
+					},
+				},
+				arrowIndicator: true,
+				autoScroll: {
+					behavior: 'auto',
+					block: autoScrollBlock,
+				},
+				liveResize: {
+					mutation: true,
+					resize: true,
+					rootElementSelector: '.woocommerce.wc-addons-wrap',
+				},
+			},
+			popperModifiers: [
+				{
+					name: 'arrow',
+					options: {
+						padding: ( {
+							popper,
+						}: {
+							popper: { width: number };
+						} ) => {
+							return {
+								// Align the arrow to the left of the popper.
+								right: popper.width - 34,
+							};
+						},
+					},
+				},
+				{
+					name: 'offset',
+					options: {
+						offset: [ 20, 20 ],
+					},
+				},
+				{
+					name: 'flip',
+					options: {
+						allowedAutoPlacements: [ 'right', 'bottom', 'top' ],
+					},
+				},
+			],
+			callbacks: {
+				onNextStep: onNextStepHandler,
+				onPreviousStep: onPreviousStepHandler,
+			},
+		},
+		steps: [
+			{
+				referenceElements: {
+					desktop: '#adminmenu a[href="admin.php?page=wc-addons"]',
```

Comment:
It looks like it won't work on mobiles because we just have not enough space. Perhaps we need to think about reducing some steps. Will discuss it with @oneillo. Thanks for shouting out!

If you wouldn't mind, I'll skip it for this particular PR.